charts/latest/csi-driver-smb/templates/csi-smb-node.yaml

```diff
@@ -60,10 +60,13 @@ spec:
             - --csi-address=$(ADDRESS)
             - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
             - --v=2
-          lifecycle:
-            preStop:
-              exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/{{ .Values.driver.name }}-reg.sock /csi/csi.sock"]
+          livenessProbe:
```

Comment:
I'm not sure if this is the same, previously it looks like the registration path and the socket would be deleted before stopping the pod, now, there's additional setup to check if the driver registered successfully.

Reply:
yep, that was the workaround, I think it's ok to remove the workaround now with registrar v2.3.0

Reply:
for safety, I added `rm -rf ...` command back